ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/NowButton.kt

```diff
@@ -30,18 +39,24 @@ private val NowButtonShape = RoundedCornerShape(
     bottomEndPercent = 50,
 )
 
+enum class NowButtonState {
+    Before, Current, After,
```

Comment:
Changed the `NowButton` from 2 possible states to 3, as we might need to scroll either up or down, depending on which way the current point in time is within the schedule.

<img width="147" alt="image" src="https://github.com/user-attachments/assets/1e1b6c8d-4c09-4839-9bef-3adc1d342ab0">